demos/continuous_batching/agentic_ai/requirements.txt

```diff
@@ -1,3 +1,4 @@
-openai-agents==0.0.19
+openai-agents==0.0.17
```

Comment:
version 0.0.19 also works fine

Reply:
I feel like 0.0.17 is safer for now. Let's make general update as a separate task.